_posts/2020-05-20-hive-acid.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: post
+title:  "Hive ACID support in Presto"
+author: Shubham Tagra, Qubole
+excerpt_separator: <!--more-->
+---
+
+Hive ACID tables are supported in Presto since the 331 release. Hive ACID
+support is an important step towards GDPR/CCPA compliance and also towards Hive3
+support given it creates ACID tables by default.
+
+In this blog we will cover the concepts of Hive ACID tables along with the
```

Comment:
you could change the whole post to "I" instead of "we" if you want.

Reply:
Definitely remove the future tense though ..

In this blog post we cover the concepts...